.gitignore

```diff
@@ -23,3 +23,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode/launch.json
```

Comment:
add `.vscode/` instead so you can add the whole configuration

Reply:
Yes! Thanks!